src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -160,21 +158,36 @@ public User getUser(String email) {
         for (Entity entity : results.asIterable()){
             try {
 
-                String idString = entity.getKey().getName();
-                UUID id = UUID.fromString(idString);
                 String user = (String) entity.getProperty("user");
-                String text = (String) entity.getProperty("text");
-                long timestamp = (long) entity.getProperty("timestamp");
-
-                Message message = new Message(id, user, text, timestamp);
+                
+                Message message = messageQuery(user, entity);
                 messages.add(message);
+
             } catch (Exception e){
-                System.err.println("Error reading message. ");
-                System.err.println(entity.toString());
-                e.printStackTrace();
+                printError(entity, e);
             }
         }
 
         return messages;
     }
+
+    public Message messageQuery(String user, Entity newEntity){
+        
```

Comment:
remove whitespace

it should be like

public void functionName() {
  content
}

without the whitespace before/after